lilypad/cli/commands/auth.py

```diff
@@ -0,0 +1,87 @@
+"""The `auth` command to run before you can trace to hosted Lilypad Hosted Cloud."""
+
+import asyncio
+import json
+import os
+import secrets
+import time
+import webbrowser
+
+import httpx
+import typer
+
+from ...server.models import DeviceCodeTable
+
+AUTH_URL = "https://your-auth-endpoint.com/auth"
+REDIRECT_PORT = 5173
+LOGIN_URL = "http://localhost:5173/auth/login"
```

Comment:
is this the right route? wouldn't this be the same as the api in production?

Reply:
1. this is the route for local
2. this need to change to be env that user sets since it will need to point to users domain

Reply:
but when they run `lilypad local` it will serve everything from port 8000 (vs. port 5173 which is the vite dev port)?

Reply:
makes sense good eye